force_wfmanager/left_side_pane/results_pane.py

```diff
@@ -0,0 +1,50 @@
+from pyface.tasks.api import TraitsDockPane
+
+from traits.api import Instance
+
+from traitsui.api import View, UItem, VGroup
+
+from force_bdss.api import IFactoryRegistryPlugin
+from force_wfmanager.central_pane.result_table import ResultTable
+from force_wfmanager.central_pane.analysis_model import AnalysisModel
+
+
+class ResultsPane(TraitsDockPane):
+    """ Side pane which contains the WorkflowSettings (Tree editor for the
+    Workflow) and the Run button """
+    id = 'force_wfmanager.results_pane'
+    name = 'Workflow'
+
+    #: Remove the possibility to close the pane
+    closable = False
+
+    #: Remove the possibility to detach the pane from the GUI
+    floatable = False
+
+    #: Remove the possibility to move the pane in the GUI
+    movable = False
+
+    #: Make the pane visible by default
+    visible = True
+
+    #: The factory registry containing all the factories
+    factory_registry = Instance(IFactoryRegistryPlugin)
+
+    #: The table displaying the results
+    result_table = Instance(ResultTable)
+
+    #: The analysis model containing the results
+    analysis_model = Instance(AnalysisModel)
+
+    traits_view = View(VGroup(
+        UItem('result_table', style='custom'),
+    ))
+
+    def __init__(self, analysis_model, *args, **kwargs):
+        self.analysis_model = analysis_model
```

Comment:
by initializing `analysis_model` before calling the Traits init machinerie, you might be breaking validation, init, etc.

I would strongly suggest going ahead with kwargs only if there is no strong reason to do it another way. Just rely on the traits constructor:

Reply:
```
def __init__(self, analysis_model, *args, **kwargs):
￼         self.analysis_model = analysis_model
         super(ResultsPane, self).__init__(*args, **kwargs)
```

should at least be:

```
def __init__(self, analysis_model, *args, **kwargs):
         super(ResultsPane, self).__init__(*args, **kwargs)
￼         self.analysis_model = analysis_model
```

Reply:
Yep, and as with the first comment we don't need init anyway!